src/routes/auth.router.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+
+const { verifyAuth } = require("../middlewares/verifyAuth");
+const AuthController = require("../controllers/auth.controller");
+
+router.post("/signup", AuthController.signup);
+router.post("/signin", AuthController.signin);
+router.post("/change-password", verifyAuth, AuthController.changePassword);
```

Comment:
You should always validate the data coming from the user end before the controller.

`express-validator`, `joi`, etc libraries are there for data validation. **It's a must**

Reply:
Resolved.